cambiatus/frontend
# Feature/new invite dashboard

## What issue does this PR close
Closes #158 

## Changes Proposed ( a list of new changes introduced by this PR)
- [x] Create new "create invite" modal on the dashboard
- [x] create new screen "public community page" that will be public and allow for both netlink and sign up redirection
- [x] Make sure to handle failed invitation requests on the public community page 
- [x] ~Add global message for when signed in (green top bar)~
- [x] change behaviour and params for the current register to accept the new invitation method
- [x] change redirect behaviour of the current sign in page, to accept redirect to the new "public community page"

## How to test
You'll need two users in other to verify. One will be the inviter the other the invited. Generate a new link from the inviter account, and open the link on another browser with the invited.

You'll need to do it on both scenarios, logged in and guest. 

**Create invite:** On dashboard, click on "INVITE YOUR FRIENDS" that exists on the balance card


### Edit
closes #202 #203
